src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import supabase from "@/supabaseClient";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+import { Schema } from "@/database_utils";
+
+interface Props {
+    text: string;
+    date: Date;
+    location: string;
+}
+
+export type ParcelOfSpecificDateAndLocation = Pick<Schema["parcels"], "collection_datetime"> & {
+    clients: Pick<
+        Schema["clients"],
+        "flagged_for_attention" | "full_name" | "address_postcode" | "delivery_instructions"
+    > | null;
+};
+
+export type DayOverviewData = {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateAndLocation[];
+};
+
+export const getCurrentDate = (date: Date, hyphen: boolean = false): string => {
```

Comment:
can we not use JS Date (toLocaleString) for this?

Reply:
Sure I updated it